lightning/src/ln/channelmanager.rs

```diff
@@ -10852,10 +10851,9 @@ macro_rules! create_offer_builder { ($self: ident, $builder: ty) => {
 	///
 	/// # Privacy
 	///
-	/// Uses [`MessageRouter`] to construct a [`BlindedMessagePath`] for the offer based on the given
-	/// `absolute_expiry` according to [`MAX_SHORT_LIVED_RELATIVE_EXPIRY`]. See those docs for
-	/// privacy implications as well as those of the parameterized [`Router`], which implements
-	/// [`MessageRouter`].
+	/// Uses the [`ChannelManager`]'s [`MessageRouter`] to construct a [`BlindedMessagePath`] for
+	/// the offer. See those docs for privacy implications as well as those of the parameterized
+	/// [`Router`], which implements [`MessageRouter`].
```

Comment:
`Router` no longer implements `MessageRouter`. This may have been missed when we decoupled them. Could you add a commit dropping that part?

Reply:
Added a separate fixup commit updating the docs in [pr3246.23](https://github.com/shaavan/rust-lightning/commits/pr3246.23)